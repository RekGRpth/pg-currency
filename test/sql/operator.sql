SELECT 'AED'::currency < 'AED'::currency;
SELECT 'AED'::currency < 'AFN'::currency;
SELECT 'AFN'::currency < 'AED'::currency;

SELECT 'AED'::currency <= 'AED'::currency;
SELECT 'AED'::currency <= 'AFN'::currency;
SELECT 'AFN'::currency <= 'AED'::currency;

SELECT 'AED'::currency = 'AED'::currency;
SELECT 'AED'::currency = 'AFN'::currency;
SELECT 'AFN'::currency = 'AED'::currency;

SELECT 'AED'::currency >= 'AED'::currency;
SELECT 'AED'::currency >= 'AFN'::currency;
SELECT 'AFN'::currency >= 'AED'::currency;

SELECT 'AED'::currency > 'AED'::currency;
SELECT 'AED'::currency > 'AFN'::currency;
SELECT 'AFN'::currency > 'AED'::currency;

SELECT 'AED'::currency <> 'AED'::currency;
SELECT 'AED'::currency <> 'AFN'::currency;
SELECT 'AFN'::currency <> 'AED'::currency;
