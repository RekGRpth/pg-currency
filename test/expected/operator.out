SELECT 'AED'::currency < 'AED'::currency;
 ?column? 
----------
 f
(1 row)

SELECT 'AED'::currency < 'AFN'::currency;
 ?column? 
----------
 t
(1 row)

SELECT 'AFN'::currency < 'AED'::currency;
 ?column? 
----------
 f
(1 row)

SELECT 'AED'::currency <= 'AED'::currency;
 ?column? 
----------
 t
(1 row)

SELECT 'AED'::currency <= 'AFN'::currency;
 ?column? 
----------
 t
(1 row)

SELECT 'AFN'::currency <= 'AED'::currency;
 ?column? 
----------
 f
(1 row)

SELECT 'AED'::currency = 'AED'::currency;
 ?column? 
----------
 t
(1 row)

SELECT 'AED'::currency = 'AFN'::currency;
 ?column? 
----------
 f
(1 row)

SELECT 'AFN'::currency = 'AED'::currency;
 ?column? 
----------
 f
(1 row)

SELECT 'AED'::currency >= 'AED'::currency;
 ?column? 
----------
 t
(1 row)

SELECT 'AED'::currency >= 'AFN'::currency;
 ?column? 
----------
 f
(1 row)

SELECT 'AFN'::currency >= 'AED'::currency;
 ?column? 
----------
 t
(1 row)

SELECT 'AED'::currency > 'AED'::currency;
 ?column? 
----------
 f
(1 row)

SELECT 'AED'::currency > 'AFN'::currency;
 ?column? 
----------
 f
(1 row)

SELECT 'AFN'::currency > 'AED'::currency;
 ?column? 
----------
 t
(1 row)

SELECT 'AED'::currency <> 'AED'::currency;
 ?column? 
----------
 f
(1 row)

SELECT 'AED'::currency <> 'AFN'::currency;
 ?column? 
----------
 t
(1 row)

SELECT 'AFN'::currency <> 'AED'::currency;
 ?column? 
----------
 t
(1 row)

